catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/gmti/MtirpbAttribute.java

```diff
@@ -26,46 +27,15 @@
 import ddf.catalog.data.impl.BasicTypes;
 
 public enum MtirpbAttribute implements NitfAttribute<Tre> {
-    DESTINATION_POINT("destinationPoint",
-            "MTI_DP",
-            tre -> GmtiTreUtility.getTreValue(tre, "MTI_DP")),
-    PATCH_NUMBER("patchNumber", "PATCH_NO", tre -> GmtiTreUtility.getTreValue(tre, "PATCH_NO")),
-    WIDE_AREA_MTI_FRAME_NUMBER("wideAreaMtiFrameNumber",
-            "WAMTI_FRAME_NO",
-            tre -> GmtiTreUtility.getTreValue(tre, "WAMTI_FRAME_NO")),
-    WIDE_AREA_MTI_BAR_NUMBER("wideAreaMtiBarNumber",
-            "WAMTI_BAR_NO",
-            tre -> GmtiTreUtility.getTreValue(tre, "WAMTI_BAR_NO")),
-    SCAN_DATE_AND_TIME("scanDateAndTime",
-            "DATIME",
-            tre -> GmtiTreUtility.getTreValue(tre, "DATIME")),
     AIRCRAFT_LOCATION("aircraftLocation",
```

Comment:
Is there a normalized taxonomy attribute for this? I would expect all of these strings to be populated by constants.